third_party/zlib/contrib/minizip/zip.c

```diff
@@ -537,8 +537,8 @@ local ZPOS64_T zip64local_SearchCentralDir(const zlib_filefunc64_32_def* pzlib_f
         break;
       }
 
-      if (uPosFound!=0)
-        break;
+    if (uPosFound!=0)
```

Comment:
Thanks for taking care of this ! Small thing, only one instance

Reply:
ack